ckanext/cnra_schema/helpers.py

```diff
@@ -0,0 +1,75 @@
+from ast import literal_eval
+
+import logging
+
+log = logging.getLogger(__name__)
+
+
+def composite_repeating_get_formatted_contact_address_dict(package_dict_field):
+    address_line1 = package_dict_field['address']
+    address_line2 = ''
+
+    address_line2 = '{0}, {1} {2} {3}'.format(package_dict_field['city'], package_dict_field['state'],
+                                              package_dict_field['postalCode'], package_dict_field['country'])
```

Comment:
Typically I've seen the country appear on the next line.
https://faq.usps.com/s/article/International-addressing-requirements